Document 'make htmllive' (#1219)
python/devguide: documentation/devguide.rst

   > .\make html

You will find the generated files in ``_build/html``.

> .. tip:: * Substitute ``htmlview`` for ``html`` to open the docs in a web browser
>            once the build completes.
>          * Substitute ``htmllive`` (Unix only) for ``html`` to automatically
>            rebuild the docs, start a local server, and reload the page in your
>            browser after you save reST files.

Reviewer: ```suggestion
.. tip:: * Substitute ``html`` with ``htmlview`` to open the docs in a web browser
           once the build completes.
         * Substitute ``html`` (Unix only) with ``htmlview`` to automatically
           rebuild the docs, start a local server, and reload the page in your
           browser after you save reST files.
```
I find this less confusing.

Another option is to just say:
```suggestion
.. tip:: * ``make htmlview`` will open the docs in a web browser
           once the build completes.
         * ``make htmllive`` (Unix only) will rebuild the docs,
            start a local server, and automatically reload the page in your
           browser when you make changes to reST files.
```

Author: Yeah, I also found "Substitute X for Y" unclear. Maybe the original idea was that plain `make` doesn't work for Windows?.

How about your first option with "Replace X with Y", or your second one and we can add Unix/Windows tabs (https://github.com/python/devguide/issues/1196)?

Reviewer: How about this?
```suggestion
.. tip:: * Substitute ``html`` with ``htmlview`` to open the docs in a web browser
           once the build completes.
         * Substitute ``html`` with ``htmllive`` to rebuild the docs,
           start a local server, and automatically reload the page in your
           browser when you make changes to reST files (Unix only).
```
I rephrased the second tip to emphasize that the automatism is in the reloading of the page when changes are made.  The "when" could also be changed to "every time" or "whenever" to further emphasize that you can keep making changes and the page will keep reloading automatically.  The "Unix only" applies to the whole tip, so it could either go at the beginning (`On Unix, ``make htmllive``...`, but this breaks symmetry with the first tip), or at the end like I did.

Author: Thanks, I went with this, with "Replace" instead of "Substitute".
